app-office/kmymoney/kmymoney-9999.ebuild

```diff
@@ -73,9 +76,28 @@ RDEPEND="${COMMON_DEPEND}
 	quotes? ( dev-perl/Finance-Quote )
 "
 
+src_prepare() {
+	kde5_src_prepare
+	if ! use test ; then
+		sed -i \
```

Comment:
kde5.eclass now has support for recursive tests removal, using KDE_TEST="forceoptional-recursive" - that rids us of the entire src_prepare ;)